src/lib/core/data-table/data-table.ts

```diff
@@ -161,7 +145,7 @@ export class CdkTable<T> implements CollectionViewer {
   }
 
   ngOnDestroy() {
-    this._onDestroy.next();
+    this._onDestroy.complete();
```

Comment:
SGTM, thanks for the context. We'll have to continue looking at this for other components where we want to unsubscribe on destroy